config/routes.rb

```diff
@@ -90,6 +90,10 @@
       end
       resources :levels, except: [:show, :edit, :update]
 
+      get 'lesson_plan' => 'lesson_plan_items#index'
+      resources :lesson_plan_milestones, except: [:index, :show]
```

Comment:
Any issues if I refactored the models to be Course::LessonPlan::Item, Course::LessonPlan::Milestone and Course::LessonPlan::Event?